fair-api-datasets-update.py

```diff
@@ -3,58 +3,66 @@
 import os
 import requests
 from datasets.diff_helper import DiffHelper
-from common.constants import BASE_HEADERS, FAIR_API_ENDPOINT, SSL_VERIFY, FAIR_URL, DRY_RUN
+from common.auth import AUTHENTICATED_HEADERS
+from common.constants import DATASETS_URL, SSL_VERIFY, FAIR_URL, DRY_RUN
 
 def dataset_url(code):
-    return f"{FAIR_API_ENDPOINT}{code}"
+    return f"{DATASETS_URL}{code}"
 
 def get_request(dataset_code):
     resp = requests.get(
-        dataset_url(dataset_code), headers=BASE_HEADERS, verify=SSL_VERIFY
+        dataset_url(dataset_code), headers=AUTHENTICATED_HEADERS, verify=SSL_VERIFY
     )
     if resp.status_code != 200:
         data = resp.json()
-        print(f'\nFailed to get dataset: Status code: {resp.status_code}, Error message: {data["error"]["message"]}')
+        print(
+            f'\nFailed to get dataset: Status code: {resp.status_code}, Error message: {data["error"]["message"]}')
         exit(1)
     return resp
 
+
 def patch_request(data):
-    dataset_code = data['catalogue']['id']
+    dataset_code = data['code']
     resp = get_request(dataset_code)
     original = resp.json()
     diff = DiffHelper.dataset_diff(original, data)
-    print (f'\nPATCH {dataset_url(dataset_code)} --data {json.dumps(diff, indent=2)}')
+    print(
+        f'\nPATCH {dataset_url(dataset_code)} --data {json.dumps(diff, indent=2)}')
```

Comment:
Is this meant to show you the curl to use? Would be nice if it was prefixed with `curl` so this is a runnable command

Reply:
I didn't write this initially but that seems to be the intention, I'll add it in.